terraform/scripts/manually-upload-certs.sh

```diff
@@ -2,8 +2,8 @@
 
 set -eu
 
-export PASSWORD_STORE_DIR=${CERT_PASSWORD_STORE_DIR}
-
+export PASSWORD_STORE_DIR=~/.paas-pass-high
```

Comment:
I don't use the alias and have it checked out at `~/creds/paas/high`, with `PASSWORD_STORE_DIR=~/creds` so that I can simply use `pass` to see all 5 password stores I have.

Only setting `PASSWORD_STORE_DIR` if it was unset would work for me.